src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -68,38 +68,8 @@ private void testInstructorResultsPage() throws Exception {
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2",
                                                                  false, "question");
-        instructorResultsPage.waitForPanelsToExpand();
-
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
         instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRubricQuestionView.html");
-
-        // Giver Recipient Question View
-        instructorResultsPage =
```

Comment:
Confirming the removal of these is because they are already being tested in another place, i.e. the unexpanded version while we fix the expected conditions